test/stylelint/extends/extends.out.css

```diff
@@ -30,6 +30,7 @@
   width: 30%;
 }
 /* Single-line comment */
+
```

Comment:
The blank line is added before @media (at-rule) because of this rule.

There is an option to turn it off when the at-rule appears after a comment but I haven't written that yet.